openedx/features/course_experience/course_tools.py

```diff
@@ -0,0 +1,37 @@
+"""
+Support for course tool plugins.
+"""
+from openedx.core.lib.api.plugins import PluginManager
+
+# Stevedore extension point namespaces
+COURSE_TOOLS_NAMESPACE = 'openedx.course_tool'
+
+
+class CourseToolsPluginManager(PluginManager):
+    """
+    Manager for all of the course tools that have been made available.
+
+    All course tabs should implement `CourseTab`.
```

Comment:
Is this a copy-and-paste comment?  Also, it would be nice to have a CourseTool class that makes it clear what each CourseTool should implement.  You could even default is_enabled to True and potentially have a generic default icon.